README.md

```diff
@@ -21,28 +34,20 @@ This project can be used to quickly setup Satellite 6.1 or 6.2 install with the
     ```
    Note: The ansible playbook run will fail and not proceed if the free space in root partition is less than the value specified in `required_root_free_space` variable in `roles/sat6repro/vars/main.yml`
 
-### Workstation requirements:
-3. Install `ansible` package in your workstation. For RHEL boxes, [access to EPEL] (https://access.redhat.com/solutions/3358) is required.
+#### On Workstation
 
-    ```console
-     # yum install -y ansible
-    ```
-4. git clone this project in your workstation.
```

Comment:
My reasonning is for the first link, they would just be clicking a link for something that is already right below